app.rb

```diff
@@ -0,0 +1,166 @@
+require './book'
+require './teacher'
+require './student'
+require './rental'
+
+class App
+  attr_reader :books, :people, :rental
+
+  def initialize
+    @books = []
+    @people = []
+    @rental = []
+  end
+
+  def run
+    show_menu
+
+    menu_list = {
+      '1' => method(:book_list),
+      '2' => method(:people_list),
+      '3' => method(:add_teacher),
+      '4' => method(:add_student),
+      '5' => method(:add_book),
+      '6' => method(:rent_book),
+      '7' => method(:rent_history)
+    }
+
+    menu = gets.chomp
+    if menu.to_i.positive? && menu.to_i <= 7
+      menu_list[menu].call
+    elsif menu == '0'
+      puts 'Thank you for using this application.'
+    else
+      puts 'Invaid operation, try again.'
+      run
+    end
+  end
```

Comment:
✅ The method to show off the menu is very organized and well presented, can easily be used by the user 👍🏻